ods_ci/tests/Tests/0100__platform/0101__deploy/0104__operators/0104__rhods_operator/0110__service_mesh.robot

```diff
@@ -17,9 +17,13 @@ ${SERVICE_MESH_OPERATOR_NS}                 openshift-operators
 ${SERVICE_MESH_OPERATOR_DEPLOYMENT_NAME}    istio-operator
 ${SERVICE_MESH_CR_NS}                       istio-system
 ${SERVICE_MESH_CR_NAME}                     data-science-smcp
-
-${IS_PRESENT}                           0
-${IS_NOT_PRESENT}                       1
+${OLM_DIR}                                  olm
+${RHODS_OSD_INSTALL_REPO}                   https://gitlab.cee.redhat.com/data-hub/olminstall.git
```

Comment:
why? this was working even without adding variable so far, no?

Reply:
No, I reviewed and if I am not wrong, no other Test Case is using olm install. these variables were demanded at execution time

Reply:
removed